universal-application-tool-0.0.1/app/views/applicant/ApplicantLayout.java

```diff
@@ -33,31 +34,56 @@
 import views.style.Styles;
 
 public class ApplicantLayout extends BaseHtmlLayout {
+
   private static final String CIVIFORM_TITLE = "CiviForm";
 
   private final ProfileUtils profileUtils;
   public final LanguageSelector languageSelector;
+  public final String supportEmail;
 
   @Inject
   public ApplicantLayout(
       ViewUtils viewUtils,
       Config configuration,
       ProfileUtils profileUtils,
-      LanguageSelector languageSelector) {
+      LanguageSelector languageSelector,
+      Config appConfig) {
     super(viewUtils, configuration);
     this.profileUtils = checkNotNull(profileUtils);
     this.languageSelector = checkNotNull(languageSelector);
+    this.supportEmail = checkNotNull(appConfig).getString("support_email_address");
```

Comment:
Question (blocking): How is appConfig different from configuration?

Reply:
the `Config` class is somewhat generic. labeling this param as `appConfig` was intended to make it very clear that this is the app-wide config object

Reply:
I meant there is another argument `configuration` in the constructor and I thought the two are the same.

Reply:
oof, that's embarrassing. i'll fix it in another PR